armi/reactor/converters/axialExpansionChanger/expansionData.py

```diff
@@ -217,15 +246,21 @@ def _setTargetComponents(self, setFuel):
             elif b.hasFlags(Flags.PLENUM) or b.hasFlags(Flags.ACLP):
```

Comment:
I can't comment above, but the content in the `if b.p.axialExpTargetComponent` block above should be replaced by a call to `_setExpansionTarget`.